src/modules/atommodel/particles/tokens/RadixMutableSupplyTokenDefinitionParticle.ts

```diff
@@ -71,38 +67,40 @@ export class RadixTokenDefinitionParticle extends RadixParticle implements Radix
     ) {
         super()
 
-        this.address = address
+        this.rri = new RRI(address, symbol)
         this.name = name
-        this.symbol = symbol
         this.description = description
         this.granularity = new RadixUInt256(granularity)
         this.iconUrl = iconUrl
         this.permissions = permissions
```

Comment:
`MutableSupplyTokenDefinitionParticle` MUST be mintable, so you need to check if mintPermissions is *not* `NONE`, else throw an error maybe?

Reply:
Java library performs the check, so I thought it prudent to follow that 🤷‍♂️

Reply:
Ok, that makes sense, I'll add it